packages/lesswrong/server/editor/mentions-notify.ts

```diff
@@ -0,0 +1,42 @@
+import * as _ from 'underscore'
+import {createNotifications} from '../notificationCallbacksHelpers'
+import {isNotificationDocument} from '../../lib/notificationTypes'
+
+export interface PingbackDocumentPartial {
+  _id: string,
+  pingbacks?: {
+    Users: string[]
+  }
+}
+
+export const notify = async (currentUser: DbUser, collectionType: string, document: PingbackDocumentPartial, oldDocument?: PingbackDocumentPartial) => {
+  const newDocPingbacks = document.pingbacks?.Users || []
+  const oldDocPingbacks = oldDocument?.pingbacks?.Users || []
```

Comment:
nit: prefer `??` when not explicitly relying on `||` to coalesce on `0`/`''`/other non-null/undefined false-y values

Reply:
👍 should we enable https://typescript-eslint.io/rules/prefer-nullish-coalescing/ ?